patches/chrome-installer-mac-signing-model.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/installer/mac/signing/model.py b/chrome/installer/mac/signing/model.py
+index 51077351e901369426b20c33e833e83dde25362f..c566446652be895bb7e7a8be1094e9dee87b91f5 100644
+--- a/chrome/installer/mac/signing/model.py
++++ b/chrome/installer/mac/signing/model.py
+@@ -295,7 +295,7 @@ class Paths(object):
+         Returns:
+             Path to the packaging directory.
+         """
+-        return os.path.join(self.input, '{} Packaging'.format(config.product))
```

Comment:
same as above, why are we adding underscores everywhere?

Reply:
I couldn't make GN call the `build/mac/notarize_dmg_pkg.py` script (as an `action`) with an argument that contained spaces (i.e. `Brave Browser Nightly Packaging`). It needs the packaging directory as the argument `--pkgdir` which failed with spaces.

Reply:
there shouldn't be any problem with spaces, you just need to quote it properly. Can you link me to the gn target where you are using it?

Reply:
It is used specifically as an argument to the `notarize_dmg_pkg.py` script here: https://github.com/brave/brave-core/pull/3725/files#diff-39842cb5e3be671a7dd386e445aca164R328